scripts/index.js

```diff
@@ -0,0 +1,195 @@
+
+const colorOrange = (event) => {
+    const currentTemp = document.querySelector("#num");
+    currentTemp.className = `orange`;
+  }; 
+
+const colorRed = (event) => {
+    const currentTemp = document.querySelector("#num");
+    currentTemp.className = `red`;
+  };
+
+// GARDEN LANDSCAPES
+
+const freezingDay = (event) => {
+    const gardenContainer = document.querySelector("#gardenContainer");
+    gardenContainer.textContent = "🧊🌨️🥶🧊";
+};
+
+const coldDay = (event) => {
+    const gardenContainer = document.querySelector("#gardenContainer");
+    gardenContainer.textContent = "🌲⛄️🍂⛸";
+};
+
+const coolDay = (event) => {
+    const gardenContainer = document.querySelector("#gardenContainer");
+    gardenContainer.textContent = "⛅🌸🌱🪁";
+};
+
+const warmDay = (event) => {
+    const gardenContainer = document.querySelector("#gardenContainer");
+    gardenContainer.textContent = "🌤🌻🌷🌼🌳🧺";
+};
+
+const hotDay = (event) => {
+    const gardenContainer = document.querySelector("#gardenContainer");
+    gardenContainer.textContent = "🌞🌴🌊🏖️🩴";
+};
+
+// SKY DROPDOWN
+
+const selectSky = (event) => {
+    const result = document.querySelector('#result');
+
+    if (event.target.value == "snowy") {
+        result.textContent = "🌨❄️☃️🌨❄️☃️";
+    } else if (event.target.value == "rainy") {
+        result.textContent = "⛈💦☔️⛈💦☔️";
+    } else if (event.target.value == "cloudy") {
+        result.textContent = "☁️ 🌫☁️ 🌫";
+    } else if (event.target.value == "sunny") {
+        result.textContent = "☀️ 🔥🌤 ☀️🔥🌤 ";
+    };
+};
+
+// Other way to write w/o registering to event handlers on line 172 -
+
+// const selectElement = document.querySelector('#skies');
+
+// selectElement.addEventListener('change', (event) => {
+//   const result = document.querySelector('#result');
+
+//   if (event.target.value == "snowy") {
+//       result.textContent = "🌨❄️☃️🌨❄️☃️";
+//   } else if (event.target.value == "rainy") {
+//       result.textContent = "⛈💦☔️⛈💦☔️";
+//   } else if (event.target.value == "cloudy") {
+//       result.textContent = "☁️ 🌫☁️ 🌫";
+//   } else if (event.target.value == "sunny") {
+//       result.textContent = "☀️ 🔥🌤 ☀️🔥🌤 ";
+//   };
+// });
+
+// OBTAIN CITY NAME
+
+const updateCityName = (event) => {
+    const cityName = document.querySelector('#city');
+    cityName.textContent = event.target.value;
+};
+
+// Rewrote example from MDN documentation into format I understood better (does not require adding to RegisterEventHandlers) -
+
+// const input = document.querySelector('input');
+// const cityName = document.querySelector('#city');
+
+// input.addEventListener('input', (event) => {
+//     cityName.textContent = event.target.value;
+// })
+
+// Original MDN:
+// input.addEventListener('input', updateValue);
+// function updateValue(e) {
+//   log.textContent = e.target.value;
+// }
+
+// RESET CITY NAME
+
+const resetCity = (event) => {
+    const cityName = document.querySelector("#city");
+    cityName.textContent = "Ada";
```

Comment:
Nice work implementing the reset. Consider also clearing the `value` attribute of this input element or setting it as the same as the `textContent`.